package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "start": "ng serve",
     "lint": "tslint \"src/**/*.ts\" and sass-lint 'src/**/*.scss' -v -q",
-    "test": "ng test --code-coverage --reporters=coverage-istanbul",
+    "test": "ng test --watch=false --code-coverage",
```

Comment:
This is not generally how it works. In local testing, `ng test` should run in watch mode and in CI env, it should run with `--watch=false`. This is taken care by the tool vendors by using environment variables, so we may skip the ``--watch=false` thing.

Reply:
Okay I am changing it.